tiledb/sm/rest/rest_client.cc

```diff
@@ -81,7 +81,8 @@ RestClient::RestClient()
     : stats_(nullptr)
     , config_(nullptr)
     , compute_tp_(nullptr)
-    , resubmit_incomplete_(true) {
+    , resubmit_incomplete_(true)
+    , memory_tracker_(nullptr) {
```

Comment:
This constructor seems fairly suspect. Is this something that's actually used?

Reply:
Or to put it another way, how many places break if we just delete this constructor?

Reply:
Ohhhh. Its just an old super non-C.41 constructor. Lets open a story to come back and fix this one.

Reply:
https://app.shortcut.com/tiledb-inc/story/41245/remove-non-c-41-constructors-from-restclient Here's the story.

Reply:
Thanks!